llvm/test/MC/X86/avx10.2-copy-32-att.s

```diff
@@ -0,0 +1,17 @@
+// RUN: llvm-mc -triple i386 --show-encoding %s | FileCheck %s
+
+// CHECK: vmovd   %xmm2, %xmm1
+// CHECK: encoding: [0x62,0xf1,0x7e,0x08,0x7e,0xca]
+          vmovd   %xmm2, %xmm1
```

Comment:
Missing memory tests.

Reply:
These are not preset in akshay's original patch.
Adding it here.